packages/mask/src/plugins/Predict/locales/en-US.json

```diff
@@ -0,0 +1,35 @@
+{
+    "plugin_predict": "Predict",
+    "plugin_azuro_bet_placed": "Bet placed",
+    "plugin_azuro_protocol": "Azuro",
+    "plugin_azuro_events": "Events",
+    "plugin_azuro_bets": "My bets",
+    "plugin_azuro_filter_all": "All",
+    "plugin_azuro_filter_active": "Active",
+    "plugin_azuro_filter_ended": "Ended",
+    "plugin_azuro_market": "Market",
+    "plugin_azuro_league": "League",
+    "plugin_azuro_sort": "Sort",
+    "plugin_azuro_status": "Status",
+    "plugin_azuro_pending": "Pending",
+    "plugin_azuro_finished": "Finished",
+    "plugin_azuro_canceled": "Canceled",
+    "plugin_azuro_redeemed": "Redeemed",
+    "plugin_azuro_redeem": "Redeem",
+    "plugin_azuro_search_events": "Search events...",
+    "plugin_azuro_place_bet": "Place bet",
+    "plugin_azuro_enter_an_amount": "Enter an amount",
+    "plugin_azuro_insufficient_amount": "Insufficient balance",
+    "plugin_azuro_share_button": "Share",
+    "plugin_azuro_share": "I just placed a sports bet using a decentralized protocol, hope I will win!\nFollow @{{account}} (mask.io) to start betting too.",
+    "plugin_azuro_share_no_official_account": "I just placed a sports bet using a decentralized protocol, hope I will win!\nWelcome to join.",
+    "plugin_azuro_success_message": "Your bet has been successfully placed.\n You can find it in the 'My bets' tab.",
+    "plugin_azuro_live": "Live",
+    "plugin_azuro_result": "Result",
+    "plugin_azuro_amount": "Amount",
+    "plugin_azuro_odd": "Odd",
+    "plugin_azuro_pick": "Pick",
+    "plugin_azuro_versus": "vs",
+    "plugin_azuro_home_team": "Home team",
+    "plugin_azuro_away_team": "Away team"
```

Comment:
You should omit `plugint_azuro_` prefix here. And don't forget to remove duplicate ones in `shared-ui/locales`

Reply:
Yes. In progress.

Reply:
Done.